packages/amplify-category-auth/src/provider-utils/awscloudformation/index.js

```diff
@@ -280,14 +306,24 @@ function getRequiredParamsForHeadlessInit(projectType, previousValues) {
     if (previousValues.authProviders.includes('www.amazon.com')) {
       requiredParams.push('amazonAppId');
     }
+    if (previousValues.authProviders.includes('appleid.apple.com')) {
```

Comment:
What's the URL like can we make sure we don't do an includes since it doesn't start with 'www'

Reply:
This is an array of [Cognito Identity Pool providers ](https://docs.aws.amazon.com/cognito/latest/developerguide/apple.html#sign-in-with-apple-cli-examples)

A few lines up you can see the facebook and google ones which, would have triggered the same warning, that's why I thought this was ok.